pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -500,10 +500,16 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 		}
 		return changes
 	}
-
+    allContainerStatusRunning := true
```

Comment:
Nit: Mind fixing the indentation here?

Reply:
I have fixed the indentation and added a unit test for this behavior.
Please help me to do a review,thanks.